src/storage/src/table/batch_table/storage_table.rs

```diff
@@ -288,6 +287,9 @@ impl<S: PkAndRowStream + Unpin> TableIter for S {
 
 /// Iterators
 impl<S: StateStore> StorageTable<S> {
+    /// The concurrency for constructing and flatten-polling the iterators.
+    const ITERATOR_CONCURRENCY: usize = 4;
```

Comment:
How should this magic number be determined?

Reply:
Just set it randomly. 😢